### PR TITLE
chore: replace yarn build with npm build for ovsx

### PR DESCRIPTION
**Pull request by svor**
Use npm to build ovsx library instead of yarn.
```
"lockfileVersion": 3
```

These changes where tested by using https://pkgs.devel.redhat.com/cgit/containers/devspaces-pluginregistry/log/?h=private-vsvydenk-npm

The resulted image is: registry-proxy.engineering.redhat.com/rh-osbs/devspaces-pluginregistry-rhel8:devspaces-3-rhel-8-containers-candidate-46330-20240828100924

**Related issue:** https://issues.redhat.com/browse/CRW-6884

### CODE REVIEW

**Review by SDawley — APPROVED** [review]
lgtm and it built successfully in brew.

File: devspaces-pluginregistry/container.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ remote_sources:
 - name: devspaces-pluginregistry
   remote_source:
     repo: https://github.com/redhat-developer/devspaces-images.git
-    ref: 593fd860980f158888b6dd19838c2dcaf4d0e8ee
-    pkg_managers: [yarn]
+    ref: 1c4e16268ea9aca3722bb6c514a91743f6ef795f
```
Comment (svor) [inline devspaces-pluginregistry/container.yaml:22]: TODO: Needs to be updated after the merge

**Comment by svor** [comment]
@SDawley could you please take a look at the last commit, I've updated `container.yaml` to use npm and `job-config.json` by removing yarn (not sure where it was used)

Last changes are built on https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=63739646

**Comment by SDawley** [comment]
@svor Changes to the container.yaml look good.
The yarn setting in the config.json determined which version of yarn was downloaded during the dashboard builds. It won't be needed once we merge https://github.com/eclipse-che/che-dashboard/pull/1170 but for now we should probably leave it.

**Pull request merged**